dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -81,13 +81,13 @@
 		#gpio-cells = <2>;
 	};
 
-	flexspi: spi@134000 {
+	flexspi: flash-controller@134000 {
 		compatible = "nxp,imx-flexspi";
 		reg = <0x134000 0x1000>;
 		interrupts = <42 0>;
 		label = "FLEXSPI";
 		#address-cells = <1>;
-		#size-cells = <0>;
+		#size-cells = <1>;
```

Comment:
Could you please help me understand this change.

Reply:
#size-cell need to be set to 1 in order to tell DTC that register size should be emitted as a 32-bit value.